web/styles/subscriptions.css

```diff
@@ -560,6 +571,31 @@ h4.user_group_setting_subsection_title {
     width: 100%;
 }
 
+.left .fa-ban,
+.right .fa-ban {
+    scale: 0.8;
+    opacity: 0.6;
+}
+
+.user-groups-list.hide-deactived-user-groups .deactivated-user-group-info-box {
+    position: absolute;
+    opacity: 0;
+    pointer-events: none;
+}
+
+.user-groups-list.hide-active-user-groups .group-row.user-group-info-box {
+    position: absolute;
+    opacity: 0;
+    pointer-events: none;
```

Comment:
I am assuming this CSS is to hide the group row as per the filter. I feel `display: none` feels more intutive solution to hide the row.